accounts/views.py

```diff
@@ -282,3 +323,18 @@ class LabsProtectedViewSet(LabsView):
 
     def get(self, request, format=None):
         return HttpResponse({"secret_information": "this is a Penn Labs protected route"})
+
+
+class MajorViewSet(generics.ListAPIView):
```

Comment:
I think `MajorView` makes more sense since this is no longer a viewset

Reply:
fixed